### PR TITLE
Fixed navigation background on osu.edu subdomains

### PR DESCRIPTION
**Pull request by LeviWald**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -11495,6 +11495,20 @@ INVERT
 
 ================================
 
+osu.edu
+ohio-state.edu
+
+CSS
+#osu_navbar {
+    background-image: linear-gradient(rgb(40, 43, 45) 0%, rgb(63, 69, 71) 100%);
+    border-bottom: 5px solid rgb(187, 0, 0);
```
Comment (LeviWald) [inline src/config/dynamic-theme-fixes.config:11504]: `height: 42px;`
[Optional] Insert the line above - refinements

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -11495,6 +11495,20 @@ INVERT
 
 ================================
 
+osu.edu
+ohio-state.edu
+
+CSS
+#osu_navbar {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11502]: I don't see anything wrong with the current styling, could you elaborate why these fixes are needed?

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -11495,6 +11495,20 @@ INVERT
...
+    background-image: linear-gradient(rgb(40, 43, 45) 0%, rgb(63, 69, 71) 100%);
+    border-bottom: 5px solid rgb(187, 0, 0);
+}
+body.sidebar-layout #page #content, #page #content {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11506]: Where can I find these elements?

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -11495,6 +11495,20 @@ INVERT
 
 ================================
 
+osu.edu
+ohio-state.edu
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11499]: Doesn't seem to be a site?

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (LeviWald) [reply src/config/dynamic-theme-fixes.config:11502]: It applies to a lot of osu.edu URLs. Examples: [engineering.osu.edu](https://engineering.osu.edu), [registrar.osu.edu](https://registrar.osu.edu), [osu.edu/map](https://www.osu.edu/map/), and the [sitemap](https://www.osu.edu/a-z-site-list.html).
Some have the navbar dark by default but the rest of the site is light: [math.osu.edu](https://math.osu.edu), [research.osu.edu](https://research.osu.edu/), and [library.osu.edu](https://library.osu.edu/). I do not know if there is a way to automatically detect this

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (LeviWald) [reply src/config/dynamic-theme-fixes.config:11506]: I originally found it on these pages: [osu.edu/findpeople](https://www.osu.edu/findpeople/) and the [2FA Form](https://buckeyelink.osu.edu/?login=true) (redirects to it).
I just recently came across the same styling change on the [sitemap](https://www.osu.edu/a-z-site-list.html), but I'm sure there's more pages I'm missing.
image (taken from sitemap)
Normally barely visible, the background image becomes much more noticeable when inverted.

I don't have a fix that'll work across most/all other pages.

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (LeviWald) [reply src/config/dynamic-theme-fixes.config:11499]: Primarily the [login page](https://buckeyelink.osu.edu/?login=true) (redirects to it) that students and staff visit daily

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11506]:
```suggestion
body.sidebar-layout #page #content,
#page #content {
```

File: src/config/dynamic-theme-fixes.config [hunk 5]
```diff
@@ -11495,6 +11495,20 @@ INVERT
...
+
+CSS
+#osu_navbar {
+    background-image: linear-gradient(rgb(40, 43, 45) 0%, rgb(63, 69, 71) 100%);
+    border-bottom: 5px solid rgb(187, 0, 0);
+}
+body.sidebar-layout #page #content, #page #content {
+    background: none;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:11507]:
```suggestion
    background-image: none;
```

**Review by Gusted — COMMENTED** [review]
Just a few changes, then it's good to go.

**Review by Gusted — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**